### ISSUE TITLE
Relative link for ssb-query is broken

### ISSUE THREAD

**Issue by enko** [ssbc/modules.scuttlebutt.nz#6]
The link at the bottom to `index.js` on https://modules.scuttlebutt.nz/server/plugins/ssb-query.html is broken. It links to https://modules.scuttlebutt.nz/server/plugins/index.js but should link to https://github.com/ssbc/ssb-query/blob/master/index.js#L26.

I think this is a gitbook issue, because it should translate the relative link to an absolute link, I searched the issues, but could not find something usefull. Any ideas?

**Comment by christianbundy** [comment]
Great catch! My first thought was to convert the links via nunjucks, but I wonder whether there's an easier way to solve the problem. Do you think it would make sense to just change the readme(s) to use absolute links to GitHub rather than relative links?

**Comment by enko** [comment]
I would expect gitbook to translate the links, but I'm not shure on how to do this. If I was tasked on resolving this issue I would complain about it at the gitbook issue tracker and ask there for a solution.

**Comment by christianbundy** [comment]
Yeah, it might be wise to at least ping them and ask if this is something they'd consider resolving. If it could be resolved as a plugin I think that would be fine too.

**Comment by ahdinosaur** [comment]
++ the links should work!

compare with http://pull-stream.github.io/ (https://github.com/pull-stream/pull-stream-docs) or http://stack.gl/packages/ (https://github.com/stackgl/packages/)

not only should relative links work (without forcing the modules to use absolute links), but i reckon we should override the h1 to be a link to the module's homepage.
